docs/user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -6,12 +6,19 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/introduction-to-pl
 
 # Introduction to plugins
 
-Strapi is built around plugins of different kinds. There are the core plugins which are essential for your Strapi application to function, and therefore cannot be deactivated. But there are also other plugins, that can either be installed by default or not, to add more options and possibilities to your Strapi application.
+Strapi is built around different types of plugins. Core plugins are essential for your Strapi application to function, and therefore cannot be deactivated. Pre-installed plugins, such as Internationalization, are installed in a default Strapi application. Additional plugins that extend and customize Strapi applications are available in the [Marketplace](../plugins/installing-plugins-via-marketplace.md).
 
 ::: note
-Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the others, non-core plugins.
+Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the other, non-core plugins.
 :::
+![Plugins settings](../assets/plugins/plugins-settings.png)
```

Comment:
Actually, since we don't have much content after the screenshot, I'm afraid readers will not see the rest of the page, so it would be better to move that screenshot at the bottom of the page (sorry 🙈)